src/chebcoefs.jl

```diff
@@ -1,24 +1,27 @@
+# FUNCTIONS FOR THE DECOMPOSITION OF THE POWER SPECTRUM ON THE BASIS OF THE CHEBYSHEV POLYNOMIALS, CENTRAL PART OF BLAST
 """
-    plan_fft(vals::AbstractArray{<:Number,N})
+    plan_fft(vals::AbstractArray{<:Number, N}, axis::Int)
 
-Create an FFTW real-to-real (R2R) transformation plan for the first axis of a given multidimensional array `vals`. 
-In practise, the `vals` array is the power spectrum P(k,χ). The first axis should then contains the wavenumbers `k`, while the second axis contains the `χ` information.
+Create an FFTW real-to-real (R2R) transformation plan for a specified axis of a given multidimensional array `vals`. 
+In practice, the `vals` array is often the power spectrum P(k,χ): the specified axis should contain the wavenumbers `k`. So if the power spectrum is given in a matrix of shape (nk, nχ), axis should be `1`. Instead, axis = `2` should be used for a matrix of shape (nχ, nk).
```

Comment:
I am confused. The chosen axis should depend only  on which axis you want to perform the FFT on, right? Sometimes it is the axis related to `z`, sometimes the axis realted to `chi`. Right?